packages/components/src/common/CloseButton/CloseButton.tsx

```diff
@@ -2,25 +2,27 @@ import clsx from "clsx";
 import React from "react";
 import Button from "../../Button";
 import CloseRoundedIcon from "../../Icons/CloseRounded";
-import { NotificationVariant } from "../Notifications/NotificationIcon";
```

Comment:
It felt weird to me that this button is so closely tied to the notification components now that it's being used somewhere else as well.